src/resources/views/landing.blade.php

```diff
@@ -28,15 +28,42 @@
 		<div class="landing-text">
 			<h1>Virtual Wellness System</h1>
 			<br>
-			<h5>The landing page</h5>
+			<h5>Enter Your Email and Password Here!</h5>
 			<br>
+				<div class="mb-4">
+                    <label for="email" class="sr-only">Email</label>
+                    <input type="text" name="email" id="email" placeholder="Your email" class="bg-gray-100 border-2 w-full p-2 rounded-lg @error('email') border-red-500 @enderror" value="{{ old('email') }}">
```

Comment:
all these helper methods are tailwind - p-2, w-full, border-2 I'm pretty sure are helper methods in that css library